lib/news_scraper/cli.rb

```diff
@@ -0,0 +1,101 @@
+require 'readline'
```

Comment:
All of this file is to give a standard logging method and give a nice output.
You'll recognize the formatting